src/Board.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import Row from "./Row.js";
+import PropTypes from "prop-types";
+
+const tableStyles = { // inline style attributes for Board
+  border: "2px solid white",
+  fontStyle: "bold"
+};
+
+function Board(props) { // destructuralisation
+  return (
+    <table style={tableStyles}>
+      <tbody>
+      {props.boardState.map(function(row, index) {
+        return (<Row row={row} key={index} />)
```

Comment:
For this usecase it is probably ok. But generaly, using array index as `key` is not good practice and can lead to buggy behaviour.

More resources: 
- https://reactjs.org/docs/lists-and-keys.html#keys
- https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
From what I've read looks like that in this case it might not be a big issue to use `index`, otherwise I could put there some unique ID based on shortid for example.